src/index.html

```diff
@@ -5,8 +5,405 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  </head>
+  <body class="body">
+    <header class="body__header header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__element active">
+            <a class="is-active">
```

Comment:
The href attribute is the most important attribute of the <a> element. Please, add it for <a> elements.